src/core/manager.h

```diff
@@ -406,6 +406,8 @@ struct Manager {
         char *switch_root;
         char *switch_root_init;
 
+        bool switching_root;
```

Comment:
why have that as a separate boolean? Why isn't `m->objective == MANAGER_SWITCH_ROOT` enough?

Reply:
`MANAGER_SWITCH_ROOT` is set before the system is going to switch root. The `switching_root` flag is set the PID1 started after switching root. Maybe, the name of the boolean flag should be changed. Do you have any better idea?

Reply:
might suffice to add a big comment here, explaining that this is set both before *and* after the switch root, in the old and the new PID 1 process.